<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>489</y>
     <width>190</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="Rmain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>601</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>731</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="LeftLabel">
       <property name="font">
        <font>
         <family>나눔바른고딕</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>LeftLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MidLabel">
       <property name="font">
        <font>
         <family>나눔바른고딕</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>MidLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RightLabel">
       <property name="font">
        <font>
         <family>나눔바른고딕</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>RightLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>731</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="LeftInfo">
       <property name="font">
        <font>
         <family>나눔고딕</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>NoAntialias</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>LeftInfo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MidInfo">
       <property name="font">
        <font>
         <family>나눔고딕</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>NoAntialias</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>MidInfo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RightInfo">
       <property name="font">
        <font>
         <family>나눔고딕</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>NoAntialias</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>RightInfo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>731</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>731</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="LeftBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>LeftBtn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MidBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>MidBtn</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RightBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>RightBtn</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>150</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bodoni MT</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>OpenKiosk</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>180</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bodoni MT</family>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Berno &amp; Lazy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>170</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bodoni MT</family>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>by</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>731</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="BackBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>BackBtn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CatBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>CatBtn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="VoiceBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>VoiceBtn</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>251</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListView" name="ItemList">
       <property name="cursor" stdset="0">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ResultLabel">
       <property name="font">
        <font>
         <family>나눔고딕 ExtraBold</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ResultLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="mighty">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1021</width>
     <height>591</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="RiceImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>RiceImage</string>
    </property>
   </widget>
   <widget class="QLabel" name="NoodleImage">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>NoodleImage</string>
    </property>
   </widget>
   <widget class="QLabel" name="MeetImage">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>MeetImage</string>
    </property>
   </widget>
   <widget class="QLabel" name="SoupImage">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>SoupImage</string>
    </property>
   </widget>
   <widget class="QLabel" name="SpicyImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>SpicyImage</string>
    </property>
   </widget>
   <widget class="QLabel" name="FriedImage">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>FriedImage</string>
    </property>
   </widget>
   <widget class="QLabel" name="OilImage">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>OilImage</string>
    </property>
   </widget>
   <widget class="QLabel" name="HotImage">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>HotImage</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>1001</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="RiceBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RiceBtn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="NoodleBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>NoodleBtn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FriedBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>FriedBtn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SpicyBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SpicyBtn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MeetBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>MeetBtn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="OilBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>460</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OilBtn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SoupBtn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>210</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SoupBtn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HotBtn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>460</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>HotBtn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CompleteBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>512</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CompleteBtn</string>
    </property>
   </widget>
  </widget>
  <zorder>Rmain</zorder>
  <zorder>mighty</zorder>
  <zorder>Background</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
